crates/annuity/src/tests.rs

```diff
@@ -8,18 +8,29 @@ fn should_calculate_emission_rewards() {
     run_test(|| {
         <Balances as Currency<AccountId>>::make_free_balance_be(&Annuity::account_id(), YEAR_1_REWARDS);
         Annuity::update_reward_per_block();
-        assert_eq!(Annuity::reward_per_block(), 12000);
+        assert_eq!(Annuity::reward_per_block(), unit(12000));
 
         <Balances as Currency<AccountId>>::make_free_balance_be(&Annuity::account_id(), YEAR_2_REWARDS);
         Annuity::update_reward_per_block();
-        assert_eq!(Annuity::reward_per_block(), 9000);
+        assert_eq!(Annuity::reward_per_block(), unit(9000));
 
         <Balances as Currency<AccountId>>::make_free_balance_be(&Annuity::account_id(), YEAR_3_REWARDS);
         Annuity::update_reward_per_block();
-        assert_eq!(Annuity::reward_per_block(), 6000);
+        assert_eq!(Annuity::reward_per_block(), unit(6000));
 
         <Balances as Currency<AccountId>>::make_free_balance_be(&Annuity::account_id(), YEAR_4_REWARDS);
         Annuity::update_reward_per_block();
-        assert_eq!(Annuity::reward_per_block(), 3000);
+        assert_eq!(Annuity::reward_per_block(), unit(3000));
+    })
+}
+
+#[test]
+fn should_set_reward_per_wrapped() {
+    run_test(|| {
+        <Balances as Currency<AccountId>>::make_free_balance_be(&Annuity::account_id(), YEAR_1_REWARDS);
+        Annuity::update_reward_per_block();
+        assert_eq!(Annuity::min_reward_per_block(), unit(12000));
```

Comment:
Can you use the constants (`YEAR_1_REWARDS`, `TotalWrapped`) rather than magic constants? Also I would prefer an integration test rather than a unit test so we can check the correctness of the `TotalWrapped` implementation